parsers/parse_FluidModelFromGid.m

```diff
@@ -252,6 +261,7 @@
 end
 out = cell2mat(out);
 fldMsh.elements = out(:, 2:end);
+fldMsh.elementIDs = out(:, 1);
```

Comment:
The same discussion as for the nodes holds for the element numbering as well, right?